eth_account/account.py

```diff
@@ -458,7 +474,10 @@ def recoverHash(self, message_hash, vrs=None, signature=None):
         return self._recover_hash(message_hash, vrs, signature)
 
     @combomethod
-    def _recover_hash(self, message_hash, vrs=None, signature=None):
+    def _recover_hash(self,
+                      message_hash: Hash32,
+                      vrs: Optional[Tuple[VRS, VRS, VRS]] = None,
+                      signature: bytes = None) -> Any:  # i/o ChecksumAddress due to decorator.
```

Comment:
Same here re. changing `Any` -> `ChecksumAddress`, and casting the return value.